kong/dao/migrations/helpers.lua

```diff
@@ -0,0 +1,96 @@
+local _M = {}
+local json_decode = require("cjson.safe").decode
```

Comment:
I have to say I don't understand why we are not following the usual pattern (in the dozens of files in this codebase) of requiring the modules first, and then declaring the new module?

```lua
local json_decode = require("cjson.safe").decode


local _M = {}
```

Reply:
> I don't understand

me neither